src/paket.references

```diff
@@ -1,3 +1,4 @@
 FSharp.Core
 Fable.Core
-Fable.PowerPack
\ No newline at end of file
+Fable.PowerPack
+SourceLink.Create.CommandLine
```

Comment:
Same here, I don't really want it to become a downstream package dependency.

Reply:
It won't. That is what `copy_local: true` is for.

Reply:
https://github.com/ctaggart/SourceLink/wiki/Paket

Reply:
Right. Thinking further on this, Fable build will include sources anyways, so maybe we don't need to include the sourcelink here?

Reply:
I don't understand what you mean by `Fable build will include sources anyway`.

Reply:
Oh right. Fable is by source. Yes, you are correct.

Reply:
But it doesn't hurt. If Visual Studio support F12 go to source at design time, it would be helpful.

Reply:
I don't think I agree... I personally don't use VS, but I think I'd like it to go to local sources in a Fable project (as in "this is what you get when you compile this dependency"). Anyone else has a preference? cc @forki @alfonsogarciacaro @MangelMaxime

Reply:
The source should be identical. Other editors should be able to add support besides VS.

Reply:
I don't use VS so I don't know.

I would also prefer going to local version of the code if possible. As this would allow to not move from the editor.

I am already having this behavior with Ionide by moment but not sure if this is a feature or because, I use it to dev the libs.

Reply:
I think "Go to definition" works for Fable libraries developed by yourself because the file path is kept in the .dll. But it doesn't work for other libraries.

I think it'd be nice to be able to "Go to definition" for Fable libraries. SourceLink is magic for me as I don't know how it works :wink: but I assume it includes the sources in the package somehow. This may look as a "duplicate" as Fable libraries should already put the code in the `fable` directory, but it shouldn't add too much to the package size (which is cached by Nuget anyways) and I guess it'll be more work to make IDEs find the `fable` folder instead of using SourceLink, already an standard.